sdk/trace/src/main/java/io/opentelemetry/sdk/trace/config/TraceConfigBuilder.java

```diff
@@ -31,39 +26,6 @@
 
   TraceConfigBuilder() {}
 
-  /**
-   * Sets the global default {@code Sampler}. It must be not {@code null} otherwise {@link #build()}
-   * will throw an exception.
-   *
-   * @param sampler the global default {@code Sampler}.
-   * @return this.
-   */
-  public TraceConfigBuilder setSampler(Sampler sampler) {
-    requireNonNull(sampler, "sampler");
-    this.sampler = sampler;
-    return this;
-  }
-
-  /**
-   * Sets the global default {@code Sampler}. It must be not {@code null} otherwise {@link #build()}
-   * will throw an exception.
-   *
-   * @param samplerRatio the global default ratio used to make decisions on {@link Span} sampling.
-   * @return this.
-   */
-  public TraceConfigBuilder setTraceIdRatioBased(double samplerRatio) {
```

Comment:
Never noticed this crazy method. While I understand the sentiment, having some branches be parentBased and some not was extremely weird!